bindings/python/native/iota_client/client.py

```diff
@@ -19,6 +19,74 @@ def __init__(self, client_config=None):
     def get_handle(self):
         return self.handle
 
+    @staticmethod
+    def create_treasury_output(amount):
```

Comment:
That's not needed, since a user can't create it in the network

Reply:
Yes so I marked them as static methods. It's a bit redundant. I will remove them in the python codes.

Reply:
Removed at [0988abc](https://github.com/iotaledger/iota.rs/pull/943/commits/0988abc98ecae2304a8c54c1655915fba895433f)